fdbserver/fdbserver.actor.cpp

```diff
@@ -1798,6 +1809,12 @@ struct CLIOptions {
 			} while (t.size() != 0);
 		}
 
+		// Sets up proxy from ENV if it is not set by arg.
+		const char* proxyENV = getenv("FDB_PROXY");
+		if (proxyENV != nullptr && !proxy.present()) {
+			proxy = proxyENV;
+		}
```

Comment:
Is there a reason to use a new env variable compared to the convention of using `HTTP_PROXY` or `HTTPS_PROXY` (e.g. https://curl.se/docs/tutorial.html)? And should we perform the same checks here as we do in the `case OPT_PROXY:` case, e.g. ensure that the value is a valid format?

Reply:
Nice catch. I should do the format check when reading the env. For the reason, I follow the convention of reading FDB_BLOB_CREDENTIALS.